### DISCUSSION TITLE
Add support for Haskell

### DISCUSSION THREAD

**Discussion by wenkokke** [emacs-tree-sitter/elisp-tree-sitter#196] [Q&A]
There is [a tree-sitter grammar for Haskell](https://github.com/tree-sitter/tree-sitter-haskell).

What would need to be done to add support for this grammar in emacs-tree-sitter?

**Comment by GTrunSec** [comment]
adding the hasekll support by yourself.
https://github.com/GTrunSec/emacs-tree-sitter/blob/master/bin/devshell.toml#L55

**Comment by ubolonton** [comment]
That grammar is not included in the bundle because there are issues with building it on GitHub Actions https://github.com/emacs-tree-sitter/tree-sitter-langs/pull/16#issuecomment-906604108.

You can try compiling it yourself and then modifying `tree-sitter-load-path`.
